src/shared/ui/tokens.css

```
@@ -25,20 +31,21 @@
+  --color-gray-600: #3a3a3a;
+  --color-gray-700: #222222;
+  /*--color-gray-bg: #191919;*/
   --color-gray-0-20: rgba(255, 255, 255, 0.2);
+  --color-gray-0-70: rgba(255, 255, 255, 0.7);
   --color-gray-500-50: rgba(92, 92, 92, 0.5);
   --color-gray-600-15: rgba(58, 58, 58, 0.15);
+  --color-gray-700-50: rgba(9, 9, 9, 0.5);
 
   /* Label / UI */
-  --color-label-strong: #000000;
+  /*--color-label-strong: #000000;
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)